### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by ericgaspar**

### CODE REVIEW

File: lib/tests.sh [hunk 1]
```diff
@@ -185,7 +185,7 @@ _VALIDATE_THAT_APP_CAN_BE_ACCESSED () {
 
             log_debug "Running curl $check_domain$curl_check_path"
 
-            # Call curl to try to access to the url of the app
+            # Call curl to try to access to the URL of the app
```
Comment (yalh76) [inline lib/tests.sh:188]:
```suggestion
            # Call cURL to try to access to the URL of the app
```

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Review by yalh76 — APPROVED** [review]

File: lib/tests.sh [hunk 2]
```diff
@@ -223,28 +223,28 @@ _VALIDATE_THAT_APP_CAN_BE_ACCESSED () {
...
-            # Check if the page title is neither the YunoHost portail or default nginx page
+            # Check if the page title is neither the YunoHost portail or default NGINX page
             if [ "$page_title" = "YunoHost Portal" ]
             then
                 log_debug "The connection attempt fall on the YunoHost portal."
                 fell_on_sso_portal=1
-                # Falling on nginx default page is an error.
-            elif echo "$page_title" | grep -q "Welcome to nginx"
+                # Falling on NGINX default page is an error.
+            elif echo "$page_title" | grep -q "Welcome to NGINX"
```
Comment (alexAubin) [inline lib/tests.sh:239]: Eeeh is it this one really a typo ? I'm thinking it's probably related to the actual content of the default nginx page ?

Comment (ericgaspar) [reply lib/tests.sh:239]: You are right! Silly me. Thanks for the checking

Comment (yalh76) [reply lib/tests.sh:239]: Fixed at https://github.com/YunoHost/package_check/pull/110

**Pull request merged**